packages/verification-vue-component/src/style/components/verification/card/_base.scss

```diff
@@ -78,6 +78,26 @@
                 + .section {
                     margin-top: 1rem;
                 }
+
+                &.retraction-note {
+                    display: flex;
+                    flex-direction: column;
+                    padding: 12px;
+                    border-radius: 8px;
+                    background-color: #e8f1fe;
+
+                    .note-label {
```

Comment:
If you hadn't used the `note-` prefix `text-transform`, `font-size` and `font-weight` would come from the `.label` defined on a higher level.

Reply:
Normally I go through the CSS in the end and refactor it but just forgot to do it here. Adjusted it